activities/tool-management/1password.md

```diff
@@ -0,0 +1,26 @@
+---
+type: resource
+---
+
+# 1Password
+
+1Password is a password manager developed by AgileBits Inc. It provides a place for users to store various passwords, software licenses, and other sensitive information in a virtual vault that is locked with a PBKDF2-guarded master password.
+
+## Using 1Password
+
+* Jitsi Meet can be reached on 'publiccodenet.1password.com'.
```

Comment:
What? This is not Jitsi meet!

Reply:
I've fixed this now. @demkodo, I don't know why this was resolved - in the future, please leave a comment on where/how you've resolved something.